src/kava-event-model.js

```diff
@@ -46,6 +46,10 @@ export const KavaEventModel: {[event: string]: KavaEvent} = {
         eventModel.targetBuffer = model.getTargetBuffer();
       }
 
+      if (model.networkConnectionType != '') {
```

Comment:
```suggestion
      if (model.networkConnectionType() !== '') {
```

Reply:
I also see that `model.getDroppedFramesRatio()` doesn't have explicit type check, please always use explicit equality checks `if (model.getDroppedFramesRatio() !== null) {`
same for `getDroppedFramesRatio` check

Reply:
I don't quite understand why. first of all I control the data in the of the networkConnectionType in the model so i don't expect it to be undefined or null and anyway undefined != '' is equal to undefined !==''